src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -379,13 +379,18 @@ Third-Party Data Acknowledge from GO device
 0x02, 0x02, 0x00, 0x04, 0x0A, 0x03
 ```
 
-#### Appendix B: Sample Message Flow for IOX-USB & IOX-RS232
+#### <a name="appendixB"></a> Appendix B: Sample Message Flow for IOX-USB & IOX-RS232
```

Comment:
Can you see if [this](https://stackoverflow.com/questions/27981247/github-markdown-same-page-link) works as a substitute to using anchor tags in the mardown?

Reply:
Yes, it works. I can change to that.